www/components/home/showcase-preview.js

```diff
@@ -14,175 +12,159 @@ const imgWidth = 330;
 const imgHeight = 185;
 const margin = 30;
 
-export default () => (
-  <MediaQueryConsumer>
-    {({ isMobile, isTablet }) => {
-      if (isTablet) {
-        return (
-          <div className="showcase-container">
-            <style jsx>{`
-              .showcase-container {
-                position: relative;
-                margin: 3rem 0 -2rem;
-                pointer-events: none;
-                line-height: 0;
-              }
-              .slides {
-                display: flex;
-                flex-wrap: wrap;
-              }
-              .slide {
-                padding: 0.5rem;
-                flex: 1 1 ${isMobile ? '50' : '33.33'}%;
-              }
-              img {
-                max-width: 100%;
-                border-radius: 5px;
-                box-shadow: 0 5px 20px rgba(0, 0, 0, 0.1);
-              }
-              .overlay {
-                position: absolute;
-                top: 0;
-                left: 0;
-                right: 0;
-                bottom: -1rem;
-                background: linear-gradient(
-                  to bottom,
-                  rgba(255, 255, 255, 0) 30%,
-                  #f6f6f6 85%
-                );
-              }
-            `}</style>
-            <div className="overlay" />
-            <div className="slides">
-              {DATA.slice(0, DATA.length - 1).map((item, i) => {
-                return (
-                  <div className="slide" key={`thumbnail-${i}`}>
-                    <img
-                      src={item.src.replace(
-                        '/showcases/',
-                        '/showcase-thumbnails/'
-                      )}
-                      alt={`Showcase ${i}`}
-                    />
-                  </div>
-                );
-              })}
+export default () => {
+  const isAmp = useAmp()
+  const Img = props => React.createElement(isAmp ? 'amp-img' : 'img', props)
```

Comment:
This can just be this, right?

```suggestion
  const Img = isAmp ? 'amp-img' : 'img'
```